docs/conf.py

```diff
@@ -52,8 +53,9 @@
 
 # General information about the project.
 project = 'Django REST Framework JSON API'
-copyright = '2015, Jerel Unruh and contributors'
-author = 'Jerel Unruh'
+year = datetime.date.today().year
```

Comment:
@mblayman Sorry for this late response. Shouldn't the copyright date not just be the current year but a range or list of years starting from when the code was first released? Or just leave it at the date of first publication?  There are [lots of opinions](https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated) around this but it seems that by advancing the copyright notice automatically you are